### ISSUE TITLE
Data Visualization Website Links

### ISSUE THREAD

**Issue by esherwoo77** [tbep-tech/tbeptools#59]
@fawda123 : I'm wondering if it makes more sense to create a common fig library from the vignettes that are developed within the tbeptools package to link to our website? E.g., I was reviewing https://tbep.org/tampa-bay-nekton-index/ , and it has the stale summary image from the [StateoftheBay repository with data only through 2022](https://github.com/tbep-tech/State-of-the-Bay/tree/master/figures). 

Since tbeptools would have the most current data updates at any given time, why not create a common fig library here that can be linked out to our website and/or presentations? I imagine you might have already thought about this, but I'm posting just to be sure ;-).

**Comment by fawda123** [comment]
Yes, long standing issue that I need to address... Of the many things I needed to update for the TBNI, I had forgotten to update the image on the SoB repo (updated now).  I'll work on this.

**Comment by fawda123** [comment]
Closing this... I've decided to use the [SoB repo](https://github.com/tbep-tech/State-of-the-Bay/tree/master/figures) as the place for a common figure library.  There are several there that aren't relevant or directly created with tbeptools, so it doesn't make sense to do it here.  I'm also working on closing the loop on links for common figures that come from other places, but aren't yet linked to the SoB repo.
